src/pages/cli/project/permissions-boundary.mdx

```diff
@@ -1,51 +1,151 @@
 
-## Set up a Permissions Boundary in a new project
+## Set up a permissions boundary in a new project
+
+To initialize a project with a permissions boundary run:
 
-To initialize a project with a Permissions Boundary run: 
 ```
 amplify init --permissions-boundary <IAM Policy ARN>
 ```
 
-## Set up a Permissions Boundary in a new environment
+## Set up a permissions boundary in a new environment
+
+When creating a new Amplify environment using `amplify env add` the permissions boundary from the current environment is automatically applied to the new environment.
 
-When creating a new Amplify environment using `amplify env add` the Permissions Boundary from the current environment is automatically applied to the new environment.
+To specify a different permissions boundary for the new environment, run:
 
-To specify a different Permissions Boundary for the new environment, run:
 ```
 amplify env add --permissions-boundary <IAM Policy ARN>
 ```
 
-To explicitly specify that the new environment should NOT have a Permissions Boundary, run:
+To explicitly specify that the new environment should NOT have a permissions boundary, run:
+
 ```
 amplify env add --permissions-boundary ''
 ```
 
-If Amplify CLI is not able to automatically apply the Permissions Boundary to the new environment and `--permissions-boundary` is not specified, it will prompt for a IAM Policy ARN as a Permissions Boundary.
+If Amplify CLI is not able to automatically apply the permissions boundary to the new environment and `--permissions-boundary` is not specified, it will prompt for a IAM Policy ARN as a permissions boundary.
 
-## Update an environment's Permissions Boundary
+## Update an environment's permissions boundary
+
+To modify the permissions boundary of the current environment, run:
 
-To modify the Permissions Boundary of the current environment, run:
 ```
 amplify env update
 ```
 
-In non-interactive shells use 
+In non-interactive shells use
+
 ```
 amplify env update --permissions-boundary <IAM Policy ARN>
 ```
 
-The IAM Permissions Boundary will be applied on the next `amplify push`.
+The IAM permissions boundary will be applied on the next `amplify push`.
+
+## Set up a permissions boundary in a cross-account Amplify project
+
+Amplify CLI cannot automatically apply the existing boundary to a new environment in a different AWS account if the `--yes` flag is used during `amplify env add`. In this case, a new permissions boundary must be specified using `amplify env add --yes --permissions-boundary <IAM Policy ARN>` or to explicitly remove the permissions boundary from the new environment use `amplify env add --yes --permissions-boundary ''`.
```

Comment:
```suggestion
Amplify CLI cannot automatically apply the existing boundary to a new environment in a different AWS account if the `--yes` flag is used during `amplify env add`. In this case, a new permissions boundary must be specified using `amplify env add --yes --permissions-boundary <iam-policy-arn>` or to explicitly remove the permissions boundary from the new environment use `amplify env add --yes --permissions-boundary ''`.
```

all lowercase and kebab-case for placeholder text

https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md#16-bracket-replacement-text-in-codeblocks